python-service/src/tier_gate_e2e_fixture.py

```
@@ -0,0 +1,14 @@
+"""Intentional E2E fixtures for Codity review and navigation tests."""
+
+from pathlib import Path
+from typing import Any
+
+
+def load_customer_report(connection: Any, customer_name: str) -> list[tuple[Any, ...]]:
+    query = f"SELECT * FROM customer_reports WHERE customer_name = '{customer_name}'"
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)